Makefile

```diff
@@ -15,11 +15,12 @@
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := provider-aws
 ADMISSION_NAME              := admission-aws
-REGISTRY                    := eu.gcr.io/gardener-project/gardener
```

Comment:
Why do you remove this variable?
I would like to be able to easily re-build the images and push them to my own OCI registry, e.g.
```bash
$ make REGISTRY=my-regeistry docker-images
```

Reply:
I have removed that change.